framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -154,7 +154,7 @@ qx.Mixin.define("qx.ui.core.MExecutable",
         // remove the old binding
         if (old != null && !old.isDisposed() && ids[property] != null)
         {
-          old.removeBinding(ids[property]);
+          this.removeBinding(ids[property]);
```

Comment:
Yes. Because the tests check the opposite direction. The should be failed *if* this is the way to resolve the problem.

While the changes fix the issues I've here, I'm not sure if it's the right way. It looks like the original qx code is either broken since 1.6 for this (legal) usecase, or I'm not seeing the point.